dietpi/dietpi-software

```diff
@@ -15140,6 +15180,22 @@ _EOF_
 			[[ -d '/var/lib/pijuice' ]] && G_EXEC rm -R /var/lib/pijuice
 
 		fi
+		
+		software_id=206 # Homer
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == -1 )); then
+			
+			Banner_Uninstalling
+			
+			sudo systemctl stop nginx
```

Comment:
this is not needed as all services will be stop on install and uninstall automatically